WBooks/LandingScreen/LandingScreenView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  LandingScreenView.swift
+//  WBooks
+//
+//  Created by Megan Cole on 08/01/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import SwiftUI
```

Comment:
Why import SwiftUI? Is this a default from Xcode 11? I guess this application runs on iOS 10, so none of SwiftUI's new things will be implemented = (